contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
> Maybe use a different dependency and change the comment to e.g.:

Why use a different dependency? Is jemalloc different from the other deps at compile time? Regardless, thanks for writing that up.